src/locker/locks/tests/index.test.js

```diff
@@ -0,0 +1,31 @@
+import createLock, { isEnabled } from '../index';
```

Comment:
There's some inconsistency in the test filename / directory: sometimes it's inside a `test` folder sometimes is alongside the file itself, e.g.: in `src/locker/index.test.js`.

We should standardize.

Reply:
Yep, I wanted to discuss this with you. I like to have a `tests` folder for each "context".
The only place I didn't do this was at the root of `src` since I thought it would look like a general test folder for all src files. 
I will create a `tests` folder at `src` root to keep it consistent with all other tests.

Reply:
I see. Well I prefer if we commit to a single strategy, what if you keep the files near each other suffixed with .test.js in all cases, like the one in src/

Reply:
Let’s ask the team tomorrow to get their opinion.